src/Translatable/Translatable.php

```diff
@@ -464,10 +457,50 @@ public function replicateWithTranslations(array $except = null)
      */
     protected function isTranslationDirty(Model $translation)
     {
-        $dirtyAttributes = $translation->getDirty();
-        unset($dirtyAttributes[$this->getLocaleKey()]);
+        $dirty = $translation->getDirty();
+
+        unset($dirty[$this->getLocaleKey()]);
+
+        if ($notEmpty = ! empty($dirty)) {
+            $original = [];
+
+            foreach ($dirty as $key => $value) {
+                $original[$key] = $translation->getOriginal($key);
+            }
+
+            config([
```

Comment:
Please don't feel offended - but I have to say it:
> What the heck!?

Ok, now more objective: why do you put these values into the config? The app config isn't a place for "poor-man"-cache.